docs/Dhiway-apis.md

```
@@ -0,0 +1,719 @@
+| 500 |  |
+
+##### Security
+
+| Security Schema | Scopes |
+| --- | --- |
+| bearerAuth | |
+
+### /api/v1/cred
+
+#### GET
+##### Summary:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)